docs/getting-started.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Getting Started
+---
+
+This is a quick introduction into project setup and our CLI. For a TypeScript quick start please read: https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html
+
+## Installation
+
+TypeScriptToLua is built using [Node.js](https://nodejs.org/) and distributed via [npm](https://www.npmjs.com/). To install it, you need to create a `package.json` file in the root of your project, containing at least `{}`. Then you can use this command to add the latest version of TypeScriptToLua to your project:
+
+```bash
+npm install -D typescript-to-lua
+```
+
+## Project setup
+
+TypeScriptToLua shares the configuration format with vanilla TypeScript. This file is called `tsconfig.json` and should be located in your project's root.
+
+Basic recommended configuration:
+
+```json title=tsconfig.json
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "lib": ["esnext"],
+    "types": [],
+    "strict": true
+  },
+  "tstl": {
+    "luaTarget": "JIT"
+  }
+}
+```
+
+Check out [Configuration](configuration.md) page for more information.
+
+## Building your project
+
+Our command line interface is called `tstl` and it works almost exactly as TypeScript's `tsc`.
+
+:::note
+Even though it's possible to install `tstl` globally, it is considered a bad practice. For for testing purposes you can run it with `npx typescript-to-lua`.
```

Comment:
I don't know if we should state this so forcefully as 'bad practice'. Maybe you could soften this a bit (and put the important part first) to:
> It is recommended you install `tstl` locally and run it from your project with `npx typescript-to-lua` to prevent conflicts between projects. However, it is also possible to install and use `tstl` globally.